src/compatTest/groovy/org/ajoberstar/gradle/git/publish/BaseCompatTest.groovy

```diff
@@ -412,6 +412,25 @@ gitPublish {
     result.task(':gitPublishPush').outcome == TaskOutcome.SUCCESS
   }
 
+  def 'is configuration cache compatible'() {
+    given:
+    buildFile << """
+plugins {
+  id 'org.ajoberstar.git-publish'
+}
+
+gitPublish {
+  repoUri = '${remote.repository.rootDir.toURI()}'
+  branch = 'gh-pages'
+  contents.from 'src'
+  sign = false
+}
+"""
+
+    expect:
+    build("gitPublishPush", "--configuration-cache")
```

Comment:
This should run the same task twice to make sure the build works when reusing the cached state.